hist/hist/src/TH1.cxx

```diff
@@ -4454,6 +4454,26 @@ Double_t TH1::GetEffectiveEntries() const
    return (s[1] ? s[0]*s[0]/s[1] : TMath::Abs(s[0]) );
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Shortcut to set the four histogram colors with a single call.
+///
+/// By default: linecolor = markercolor = fillcolor = -1
+/// If a color is < 0 this method does not change the corresponding color if positive or null it set the color.
+///
+/// For instance:
+/// ~~~ {.cpp}
+/// h->SetColor(kRed, kRed);
+/// ~~~
+/// will set the line color and the marker color to red.
+
+void TH1::SetColor(Color_t linecolor, Color_t markercolor, Color_t fillcolor)
```

Comment:
May be one should call it `SetColors` or `SetHistColors`?

Reply:
As it is a TH1 method it is quite obvious it is a histogram method so there is no need to add "Hist" in the name (I checked that none of the TH1 methods have "Hist" in their name). However, adding an "s" is a good idea.